specification/attributes/unit_format.md

```diff
@@ -0,0 +1,92 @@
+# Unit Format
+
+Cost and utilization data frequently captures data measured in units related to data size, count, time, and other dimensions. The Unit Format attribute provides a standard for expressing units of measure in columns appearing in billing data.
+
+All columns defined in FOCUS specifying Unit Format as a value format MUST follow the requirements listed below.
+
+## Attribute ID
+
+UnitFormat
+
+## Attribute Name
+
+Unit Format
+
+## Description
+
+Indicates standards for expressing units of measure in columns appearing in billing data.
+
+## Requirements
+
+* Units SHOULD be expressed as a single unit of measure adhering to one of the following three formats.
+  * `<plural-units>` - "GB", "Seconds"
+  * `<singular-unit>-<plural-time-units>` - "GB-Hours", "MB-Days"
+  * `<plural-units>/<singular-time-unit>` - "GB/Hour", "PB/Day"
+* Units MAY be expressed with a unit quantity or time interval.  If a unit quantity or time interval is used, the unit quantity or time interval MUST be expressed as a whole number.  The following formats are valid:
+  * `<quantity> <plural-units>` - "1000 Tokens", "1000 Characters"
+  * `<plural-units>/<interval> <plural-time-units>` - "Units/3 Months"
+* Unit values and components of columns using the Unit Format MUST use a capitalization scheme that is consistent with the capitalization scheme used in this attribute, if that term is listed in this section. For example, a value of "gigabyte-seconds" would not be compliant with this specification as the terms "gigabyte" and "second" are listed in this section with the appropriate capitalization.  If the unit is not listed in the table, it is to be used over a functional equivalent with similar meaning with the same capitalization scheme.
+* Units SHOULD be composed of the list of recommended units listed in this section, unless the unit value covers a dimension not listed in the recommended unit set, or if the unit covers a count-based unit distinct from recommended values in the count dimension listed in this section.  
+
+### Data Size Unit Names
+
+Data size MUST be abbreviated using the following standard abbreviations.  Each data size abbreviation can be considered both the singular and plural form of the unit.  For example, "GB" is both the singular and plural form of the unit "GB".  The following table lists the valid data size units.
+
+| Data size in bits    | Data size in bytes    |
```

Comment:
Is this an _abbreviation_ table, or a _valid data size_ table?  The previous paragraph implies both.  Can a provider list their data in bytes?

Reply:
Based on previous conversations, do we want them to provide data in bytes? I want to say this came up in the practitioner squad that something that small isn't useful. For the record, I'm fine either way in 1.0.